packages/cactus-plugin-bungee/src/test/typescript/fabric-contracts/simple-asset/chaincode-typescript/.gitignore

```diff
@@ -0,0 +1,16 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Dependency directories
+node_modules/
+jspm_packages/
+package-lock.json
+
+# Compiled TypeScript files
+dist
+
```

Comment:
@eduv09 I'm guessing that this file can be removed because it won't be used at all in the test automation context. Please either remove it or just advise why it is necessary and if there is a legit reason for it to remain then I'm totally OK with that too we just have to have it documented on the purpose of leaving it under version control.

Reply:
I believe this chaincode is used to test BUNGEE

Reply:
Oh, you mean the lint, specifically. Yes, strictly speaking we just need the chaincode.